ui/components/src/TimeSeriesTooltip/utils.ts

```diff
@@ -33,20 +33,20 @@ export function assembleTransform(
 
   const cursorPaddingX = 32;
   const cursorPaddingY = 16;
-  const x = mousePos.viewport.x;
-  let y = mousePos.viewport.y + cursorPaddingY;
+  const x = mousePos.page.x;
+  let y = mousePos.page.y + cursorPaddingY;
 
```

Comment:
Summarizing to make sure I understand the problem: The issue is that the tooltip is in a portal that's absolutely positioned relative to the `body`. Previously, we were setting the positioning based on the viewport position of a panel, which will not work as expected when scrolled down the page. The fix positions the tooltip based on the position of the panel within the page. 

It might be worth putting a tiny comment somewhere in here for future us about this. Maybe something quick like "Tooltip is located in a `Portal` attached to the `body` and absolutely positioned."